react-components/src/desktop/view/Notifications.js

```diff
@@ -157,18 +158,17 @@ class Notifications extends Component {
 
         return (
             <React.Fragment>
-                <NotificationIcon
-                    id={build(ids.DESKTOP, ids.NOTIFICATION_BUTTON)}
-                    className={classes.menuIcon}
-                    onClick={this.handleNotificationsClick}
-                    ref={this.notificationBtn}
-                />
-                {unSeenCount !== 0 &&
-                <span id='notifyCount'
-                      className={classes.unSeenCount}>
-                        {unSeenCount}
-                    </span>
-                }
+                <Badge badgeContent={unSeenCount} style={{
+                    margin: '0',
+                    right: '30px'
```

Comment:
Since this file is already using `withStyles` maybe this bit of styling should go in the styles file as well?

Reply:
👍